src/components/AgreementLineSections/FormInfo/FormInfo.test.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import '@folio/stripes-erm-components/test/jest/__mock__';
+import { TestForm, renderWithIntl } from '@folio/stripes-erm-components/test/jest/helpers';
+import { Datepicker, Checkbox } from '@folio/stripes-testing';
+import translationsProperties from '../../../../test/helpers';
+import FormInfo from './FormInfo';
+import { data, handlers, initialValues, values } from './testResources';
+
+const onSubmit = jest.fn();
+const isSuppressFromDiscoveryEnabled = jest.fn(_info => true);
+
+  let renderComponent;
+  describe('FormInfo', () => {
+    describe('with no initial values', () => {
+      beforeEach(() => {
+        renderComponent = renderWithIntl(
+          <TestForm data={data} onSubmit={onSubmit}>
+            <FormInfo
+              isSuppressFromDiscoveryEnabled={isSuppressFromDiscoveryEnabled}
+            />
+          </TestForm>, translationsProperties
+        );
+      });
+
+      test('renders Description field', () => {
+        const { getByRole } = renderComponent;
+        expect(getByRole('textbox', { name: 'Description' }));
+      });
+
+      test('renders Note field', () => {
+        const { getByRole } = renderComponent;
+        expect(getByRole('textbox', { name: 'Note' }));
+      });
+
+      test('renders Active from field', () => {
+        const { getByRole } = renderComponent;
+        expect(getByRole('textbox', { name: 'Active from' }));
+      });
+
+      test('renders Active to field', () => {
+        const { getByRole } = renderComponent;
+        expect(getByRole('textbox', { name: 'Active to' }));
+      });
+
+      test('renders Suppress from discovery field', () => {
+        const { getByRole } = renderComponent;
+        expect(getByRole('checkbox', { name: 'Suppress from discovery' }));
+      });
+
+      test('renders Active from Datepicker by id', async () => {
+        await Datepicker({ id: 'agreement-line-active-from' }).exists();
+      });
+
+      test('renders Active to Datepicker by id', async () => {
+        await Datepicker({ id: 'agreement-line-active-to' }).exists();
+      });
+
+      test('renders Suppress from discovery Checkbox by id', async () => {
+        await Checkbox({ id: 'agreement-line-suppress-from-discovery' }).exists();
+      });
+
+      test('renders the Description field by id', () => {
+        const { getByTestId } = renderComponent;
+        expect(getByTestId('line-description')).toBeInTheDocument();
+      });
+
+      test('renders the Description field by id', () => {
+        const { getByTestId } = renderComponent;
+        expect(getByTestId('line-note')).toBeInTheDocument();
+      });
```

Comment:
Both the tests have the same decription.

Also, could you change this to use the getByRole textBox as you have used in the other tests? That way you can remove the data-testid passed to those Fields as well

Reply:
I have already used getByRole textbox for 'description' and 'note' in lines 25-33.